src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -141,6 +160,21 @@ export const NavBar: React.FC<NavBarPropsType> = ({ hasBackButton = true }) => {
 
   const [isRadioPlaying, setIsRadioPlaying] = useState(false);
   const { volume, setVolume } = useRadio(isRadioPlaying, sound);
+  const handleMute = useCallback(
+    (volume: number) => (volume !== 0 ? 0 : 100),
+    []
+  );
+  const toggleMute = useCallback(() => setVolume(handleMute), [
+    handleMute,
+    setVolume,
+  ]);
+
+  const volumeControlClassname = classNames(
+    "NavBar__menu--link NavBar__menu--volume",
+    {
+      "NavBar__menu--link NavBar__menu--volume--muted": !volume,
```

Comment:
"NavBar__menu--link" is already there by default. Looks like you don't need that.
```suggestion
      "NavBar__menu--volume--muted": !volume,
```